lua/yabs/outputs/buffer.lua

```diff
@@ -18,28 +21,25 @@ local function make_scratch_buffer(height, position)
   return vim.fn.bufnr()
 end
 
-local function append_to_buffer(buffer, lines)
```

Comment:
What's the rationale for deleting this? I know it's just one line, but I put it in a separate function on purpose because without it it's pretty unclear what `vim.api.nvim_buf_set_lines()` does in this case.

Reply:
I like the separation, but in this case it looks a little too explicit to me:

```lua
local function append_to_buffer(line)
  vim.api.nvim_buf_set_lines(bufnr, -2, -2, false, { line })
end

local function on_read(error, data)
  append_to_buffer(error and error or data)
end
```

But how about to just change `on_read` name into `append_to_buffer`?

Reply:
That sounds like a reasonable solution, particularly now that the function will just be that line instead of two.